src/components/IOUConfirmationList.js

```diff
@@ -322,7 +319,25 @@ class IOUConfirmationList extends Component {
                             options={this.getSplitOrRequestOptions()}
                         />
                     )}
-            />
+            >
+                <MenuItemWithTopDescription
+                    shouldShowRightIcon
+                    title={formattedAmount}
+                    description={this.props.translate('iou.amount')}
+                    interactive={false} // This is so the menu item's background doesn't change color on hover
+                    onPress={() => this.props.navigateToStep(0)} // Index 0 here because the amount step is always the first step.
```

Comment:
NAB: I know i just asked for a comment to be more descriptive, but I'm not sure that this comment is necessary.

Reply:
Sounds good!